docs/source/xrpl.utils.txn_parser.rst

```diff
@@ -0,0 +1,37 @@
+xrpl.utils.txn\_parser package
```

Comment:
These two docs pages should actually both just contain `:orphan:` - since you're exporting the txn parser stuff at the `xrpl.utils` level, the docs will appear there already, and you don't need separate pages.

Sorry for the confusion.